docs/using_lagoon/lagoon_yml.md

```diff
@@ -260,3 +260,42 @@ Each definition is keyed by a unique name (`secrets` and `logs-db-secrets` in th
 * `path` - the path to the yaml file
 * `command` - can either be `create` or `apply`, depending on if you like to run `kubectl create -f [yamlfile]` or `kubectl apply -f [yamlfile]`
 * `ignore_error` - either `true` or `false` (default), this allows you to instruct the lagoon build script to ignore any errors that might are returned during running the command. (This can be useful to handle the case where you want to run `create` during every build, so that eventual configurations are created, but don't want to fail if they already exist).
+
+#### `private-registries`
```

Comment:
can we also call this `container-registries`? then it will be clean through the whole system, also this technically can be used for non-dockerhub but public registries, so using `container-registries` would fit even more :)